src/securityLogs/index.ts

```diff
@@ -0,0 +1,186 @@
+import { CrowdinApi, ResponseList, isOptionalNumber, ResponseObject } from '../core';
+
+export class SecurityLogs extends CrowdinApi {
+    /**
+     * @param options optional parameters for the request
+     * @see https://developer.crowdin.com/api/v2/#operation/api.users.security-logs.getMany
+     */
+    listUserSecurityLogs(
+        userId: number,
+        options?: SecurityLogsModel.SecurtiyOptions,
+    ): Promise<ResponseList<SecurityLogsModel.Log>>;
+    /**
+     * @param userId User identifier
+     * @param limit maximum number of items to retrieve (default 25)
+     * @param offset starting offset in the collection (default 0)
+     * @param event
+     * @param createdAfter
+     * @param createdBefore
+     * @deprecated optional parameters should be passed through an object
+     * @see https://developer.crowdin.com/api/v2/#operation/api.users.security-logs.getMany
+     */
+    listUserSecurityLogs(
+        userId: number,
+        limit?: number,
+        offset?: number,
+        event?: SecurityLogsModel.Event,
+        createdAfter?: string,
+        createdBefore?: string,
+        ipAddress?: string,
+    ): Promise<ResponseList<SecurityLogsModel.Log>>;
+    listUserSecurityLogs(
+        userId: number,
+        options?: number | SecurityLogsModel.SecurtiyOptions,
+        deprecatedoffset?: number,
+        deprecatedevent?: SecurityLogsModel.Event,
+        deprecatedcreatedAfter?: string,
+        deprecatedcreatedBefore?: string,
+        deprecatedipAddress?: string,
+    ): Promise<ResponseList<SecurityLogsModel.Log>> {
```

Comment:
we don't need all of these deprecations in new methods. Just create a single `listUserSecurityLogs` method that accepts an object with options